iclr2017/process/commentProcess.js

```diff
@@ -67,12 +67,36 @@ function(){
       return or3client.or3request(or3client.grpUrl+'?id=ICLR.cc/2017/pcs',{},'GET',token)
       .then(result=>{
         var pcs = result.groups[0].members;
+
+        console.log('pcs before filter: '+pcs);
+
+        var workshopIdx = pcs.indexOf('ICLR.cc/2017/workshop');
+        if(workshopIdx>-1){
+          pcs.splice(workshopIdx,1);
+        }
+        var conferenceIdx = pcs.indexOf('ICLR.cc/2017/conference');
+        if(conferenceIdx>-1){
+          pcs.splice(conferenceIdx,1);
+        }
+
         var signatureIdx = pcs.indexOf(note.signatures[0]);
-        console.log('pcs before filter:',pcs);
         if(signatureIdx>-1){
           pcs.splice(signatureIdx,1);
-        };
-        console.log('pcs after filter:',pcs);
+        }
+
+        //if any member of the PCs is an author, remove that PC from email recipients
+        for(var i=0; i<origNote.content.authorids.length; i++){
+          var currentAuth = origNote.content.authorids[i];
+          var pcIdx = pcs.indexOf(currentAuth);
```

Comment:
Ok. if one of the PCs uses a different email address, should I change the authorid in the paper? or should I add the email address to the PC group?